autotest/gcore/vrt_read.py

```diff
@@ -1450,6 +1450,10 @@ def test_vrt_protocol():
     geotransform = ds.GetGeoTransform()
     assert geotransform == (0.0, 1.0, 0.0, 10.0, 0.0, -0.5)
 
+    # test #7282
+    ds = gdal.Open("vrt://data/byte_with_ovr.tif?ovr=2")
+    assert ds.RasterXSize == 5
```

Comment:
ovr=2 would be the 3rd overview level. There's a max mechanism that limits to the max overview level. Changing the test to test for ovr=0 which is the first overview level
```suggestion
    ds = gdal.Open("vrt://data/byte_with_ovr.tif?ovr=0")
    assert ds.RasterXSize == 10
```

Reply:
oops, thank you